### PR TITLE
change index.html template file to get a better show in a table 

### PR DESCRIPTION
**Pull request by ranms25**
table allows for us easy comparison and visualization of the different organization types and their respective numbers.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -2,14 +2,30 @@
 
 <h1 class="title">{{title}}</h1>
 
+<table>
+    <thead>
+        <tr>
+            <th><center>Organization Types&nbsp</center></th>
+            <th><center>&nbspNumber of Organizations</center></th>
```
Comment (szabgab) [inline templates/index.html:9]: Let's not use &nbsp and I think Bulma has some class to center a table text and we don't need to use the <center> tag. Please check.

File: templates/index.html [hunk 2]
```diff
@@ -2,14 +2,30 @@
 
 <h1 class="title">{{title}}</h1>
 
+<table>
```
Comment (szabgab) [inline templates/index.html:5]: As far as I know you need to add some class to the table in Bulma to look good.

File: templates/index.html [hunk 3]
```diff
@@ -2,14 +2,30 @@
...
+    </thead>
+    <tbody>
+        {% for org_type, display_name in org_types.items() %}
+            <tr>
+                <td><a href="/open-source-by-organizations/{{org_type}}">{{ display_name }}</a></td>
+                <td><center>{{ stats.by_type[org_type] }}</center></td>
+            </tr>
+            <tr>
+                <td colspan="2">&nbsp;</td> <!-- Add a row with an empty cell that spans both columns to create spacing between rows -->
+            </tr>
```
Comment (szabgab) [inline templates/index.html:23]: I bet there is some way with Bulma to do this instead of additional table-row.

Comment (ranms25) [reply templates/index.html:17]: Hope it's better

**Comment by szabgab** [comment]
Thanks. Let's see how does it look like on the site.

**Comment by ranms25** [comment]
> Thanks. Let's see how does it look like on the site.

you are welcome! I think it looks good 🙂
And to my delight, everything seems to work as before

**Pull request merged**